src/lsp/cobol_lsp/lsp_completion.ml

```diff
@@ -243,16 +251,21 @@ let map_completion_items ~(range:Range.t) ~case ~group ~filename comp_entries =
           let typ = approx_type_to_string typ in
           completion_item_create
             ~priority_sort:(if is_valid then 2 else 1)
```

Comment:
Could you name the Integer literals you use for priority sort with a series of `let-bindings` at the top of this module? I think that'd make the selected priorities a bit clearer.